example_external_provider_plugin/plugin.go

```diff
@@ -0,0 +1,215 @@
+package main
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/protobuf/types/known/timestamppb"
+
+	provider_service "github.com/openclarity/vmclarity/runtime_scan/pkg/provider/grpc/proto"
+
+	"github.com/openclarity/vmclarity/api/models"
+)
+
+type Provider struct {
+	provider_service.UnimplementedProviderServer
+}
+
+func (p *Provider) RunScanningJob(ctx context.Context, params *provider_service.RunScanningJobParams) (*provider_service.RunScanningJobResult, error) {
+	// Here need to run the VMClarity CLI and return the instance that is running the CLI.
```

Comment:
Here need to run the instance that runs the VMClarity CLI and to return the details of that instance